index.html

```diff
@@ -2517,26 +2513,68 @@ <h3>
         <dl>
           <dt>@context</dt>
           <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
+The value of the <code>@context</code> property MUST be exactly one of the following:
+
+<ul>
+  <li>The string <code>https://www.w3.org/ns/did/v1</code>. 
+    <pre class="example">
+      {
+        "@context": "https://www.w3.org/ns/did/v1",
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array of one or more elements,
+    where the value types of the members of the array
+    MUST be either a string or an object and the first element is
+    the string <code>https://www.w3.org/ns/did/v1</code>.
+
+    <pre class="example">
+      {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://example.com/blockchain-identity/v1"
+        ],
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array where the first value is
+    <code>https://www.w3.org/ns/did/v1</code>, and the final value is an object.
+
+    <pre class="example">
+      {
+        "@context": [
+          "https://www.w3.org/ns/did/v1",
+          "https://example.com/blockchain-identity/v1",
+          {
+            "@base": "did:example:123"
```

Comment:
Hmm... this is a problem due to this section: https://w3c.github.io/did-core/#relative-did-urls

We may have to note that different representations may impose their own Base URI resolution mechanisms (like JSON-LD) and so those run after the generalized Base URI algorithm is run. I think that's the behavior we want. 

Otherwise, this statement above is going to be ignored... and honestly, I'd argue that the use of `@base` is something we do not want to suggest that people do. If we're going to have examples like this in here, we need to say that reprepsentation-specific base processing happens after the Base URI establishment in Section 3.2.5 today.

Reply:
I don't think there is anything wrong with using `@base`, I have written tests that show that it works fine for DID Document's and VCs, and per the JSON-LD spec, it is allowed... https://json-ld.org/spec/latest/json-ld/#using-the-document-base-as-the-default-vocabulary

did:ion,  did:elem, and our implementation did:key all use `@base`... in my opinion, it makes JSON-LD DID Document representations much more aesthetically pleasing, and it produces identical N-Quads to the much more repetitive fully qualified URI representations.

`@base` does not get ignored which you compact a JSON-LD DID Document to n-quads using off the shelf tooling.

Here is an example which you can explore using the JSON-LD Playground https://json-ld.org/playground/:

```
{
  "@context": [
    "https://www.w3.org/ns/did/v1",
    {
      "@base": "did:key:z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"
    }
  ],
  "id": "did:key:z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP",
  "publicKey": [
    {
      "id": "#z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP",
      "type": "Ed25519VerificationKey2018",
      "controller": "did:key:z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP",
      "publicKeyBase58": "dbDmZLTWuEYYZNHFLKLoRkEX4sZykkSLNQLXvMUyMB1"
    }
  ],
  "authentication": ["#z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"],
  "assertionMethod": ["#z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"],
  "capabilityDelegation": ["#z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"],
  "capabilityInvocation": ["#z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP"],
  "keyAgreement": [
    {
      "id": "#z6LScqmY9kirLuY22G6CuqBjuMpoqtgWk7bahWjuxFw5xH6G",
      "type": "X25519KeyAgreementKey2019",
      "controller": "did:key:z6Mkf5rGMoatrSj1f4CyvuHBeXJELe9RPdzo2PKGNCKVtZxP",
      "publicKeyBase58": "2AbNdSuzFSpGvsiSPBfnamcKzk9Q3WRRpY2EToHZEuKW"
    }
  ]
}
```